projects/kit/components/slider/slider.style.less

```diff
@@ -130,6 +125,8 @@
 }
 
 :host {
+    --tui-ticks-thickness: @ticks-thickness;
```

Comment:
Let's use `--t-tick-thickness` (we use `--tui-***`-prefix for public variables and `--t-**` for private API).
I think `--t-tick-thickness` should be private one (for the 1st iteration, at least)

Reply:
\+ use `--t-tick-thickness` everywhere inside this `less`-file (instead of static `less`-variable `@ticks-thickness`)